src/parser/jobs/whm/modules/Lilies.tsx

```diff
@@ -0,0 +1,159 @@
+import {Plural, Trans} from '@lingui/react'
+import Color from 'color'
+import {DataLink} from 'components/ui/DbLink'
+import {ActionKey} from 'data/ACTIONS'
+import {Event} from 'event'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {CounterGauge, Gauge as CoreGauge, TimerGauge} from 'parser/core/modules/Gauge'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const LILY_MAX_STACKS = 3
+const LILY_INTERVAL_600 = 30000
+const LILY_INTERVAL_610 = 20000
+
+const MISERY_COST = 3
+
+const GRAPH_DISPLAY_SETTINGS = {
+	LILY_GAUGE_FADE: 0.25,
+	LILY_TIMER_FADE: 0.75,
+	BLOODLILY_GAUGE_FADE: 0.5,
+	LILY_GAUGE_COLOR: Color('#4f73b5'),
+	BLOODLILY_GAUGE_COLOR: Color('#b52d6c'),
+}
+
+export class Lilies extends CoreGauge {
+
```

Comment:
Best to add a handle in here, so that modules don't clash. In this case the handle and i18n keys diverge a bit from the class name but eh. Gauge works better than lily/lilies anyway. There's only one gauge for WHM iirc so it doesn't really matter, could even rename the file/class to just Gauge I guess if you cared.
```suggestion
	static override handle = 'gauge'
```